dracut/parse-anaconda-options.sh

```diff
@@ -45,7 +45,8 @@ set_http_header "X-Anaconda-System-Release" "$product"
 warn_renamed_arg() {
     local arg=""
     arg="$(getarg $1)" && warn "'$1=$arg'" && \
-        warn "$1 has been deprecated and will be removed. Please use $2 instead."
+        warn "$1 has been deprecated same as all Anaconda boot arguments without 'inst.' prefix. \
+They will be removed with a future major release. Please use $2 instead."
```

Comment:
Let's try to reuse as much as possible?

_$1 has been deprecated. All usage of Anaconda boot arguments without 'inst.' prefix has been deprecated and will be removed in a future major release. Please use $2 instead._

It's a bit trickier since this is shown also for options that do not fit the pattern "just add inst. in front": `kssendsn` -> `inst.ks.sendsn`. But let's not sweat it, the sentence is still true.